### ISSUE TITLE
Recipes  missing

### ISSUE THREAD

**Issue by MCEventHorizon** [Malorolam/LootBags#135]
In 1.12.2 version of the mod there are no 2x2 recipes to craft better bags or one better to create 4 lower tier bags.

**Comment by MCEventHorizon** [comment]
Problem for me with that is that it isn't dynamic. I made a UHC modpack and I was hoping that players could combine the bags on the move instead of placing bag storage. And making the recipes is difficult cause it doesn't recognize if the bag was opened. Would it be possible for a config file that turns on/off crafting?

**Comment by Malorolam** [comment]
No, because of how Minecraft does crafting recipes now, I can't dynamically create recipes the way they used to work.  
The unopened bags have no NBT data.

**Comment by MCEventHorizon** [comment]
What about a portable bag storage, like a backpack?

**Comment by Malorolam** [comment]
Possibly, I've been messing around with UUIDs to solve dupe glitch problems, so if I get a handle on using them properly I'll probably add a portable storage, probably based off of ender chest/pouches.
